bitbucket/resource_group.go

```diff
@@ -61,6 +79,12 @@ func resourceGroupRead(d *schema.ResourceData, m interface{}) error {
 	return fmt.Errorf("unable to find a matching group %s", groupName)
 }
 
+func resourceGroupUpdate(d *schema.ResourceData, m interface{}) error {
```

Comment:
why even adding the update method? It's not required

Reply:
As mentioned in the inline comments:

> The only attribute in the schema that does not have "ForceNew: true" is "import_if_exists",
> so we are not actually updating any groups in Bitbucket, we just need to read and return.

When the forceNew is not set for import_if_exists, we can the following error without the update function implemented. This is because there's no API to update the group attribute. 
```
1 error occurred:
                * resource bitbucketserver_group: No Update defined, must set ForceNew on: []string{"import_if_exists"}
```